src/analysis/calculus/deriv.lean

```diff
@@ -368,14 +368,14 @@ by { unfold deriv_within, rw fderiv_within_inter ht hs }
 section congr
 /-! ### Congruence properties of derivatives -/
 
-theorem has_deriv_at_filter_congr_of_mem_sets
-  (hx : f₀ x = f₁ x) (h₀ : ∀ᶠ x in L, f₀ x = f₁ x) (h₁ : f₀' = f₁') :
+theorem has_deriv_at_filter_congr_of_eventually_eq
```

Comment:
What do you think about `filter.eventually_eq.has_deriv_at_filter_congr`?

Reply:
That's a good idea. Done